src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+$day-height: 100px;
+$day-width: 101px;
```

Comment:
Consider using a more descriptive variable name for the width of the calendar cells, such as $calendar-cell-width. Also, since the task description mentions a 1px gap, the width should be 100px to account for the gap.